integrations/viem-v2/package.json

```diff
@@ -59,7 +59,11 @@
 	},
 	"devDependencies": {
 		"@biomejs/biome": "^1.7.0",
+		"@nomicfoundation/hardhat-viem": "~2.0.0",
```

Comment:
whats the reason to fix it to patch version?
(beside that they never released a minor version)

Reply:
We're not using hardhat-viem in this package at all, it's just to get the code example in README.md to work.

But yea, I should probably set it to ^2.0.0, but it's the default that came with `pnpm add -D '@nomicfoundation/hardhat-viem'`